RiotSharp/RiotApi.cs

```diff
@@ -301,106 +302,70 @@ public List<MasteryPage> GetMasteryPages(Region region, long summonerId)
         #endregion
 
         #region League
-        public Dictionary<long, List<League>> GetLeagues(Region region, List<long> summonerIds)
+        public List<League> GetLeagues(Region region, long summonerId)
         {
-            var dict = new Dictionary<long, List<League>>();
-            foreach (var grp in MakeGroups(summonerIds, MaxNrLeagues))
-            {
-                var json = requester.CreateGetRequest(
-                    string.Format(LeagueRootUrl, region.ToString()) +
-                        string.Format(LeagueBySummonerUrl, Util.BuildIdsString(grp)) + LeagueEntryUrl,
+                var json = requester.CreateGetRequest(LeagueRootUrl +
+                        string.Format(LeagueBySummonerUrl, summonerId),
                     region);
-                var subDict = JsonConvert.DeserializeObject<Dictionary<long, List<League>>>(json);
-                foreach (var child in subDict)
-                {
-                    dict.Add(child.Key, child.Value);
-                }
-            }
-            return dict;
+                var list = JsonConvert.DeserializeObject<List<League>>(json);
+            return list;
         }
  
-        public async Task<Dictionary<long, List<League>>> GetLeaguesAsync(Region region, List<long> summonerIds)
+        public async Task<List<League>> GetLeaguesAsync(Region region, long summonerId)
         {
-            var tasks = MakeGroups(summonerIds, MaxNrLeagues).Select(
-                grp => requester.CreateGetRequestAsync(
-                    string.Format(LeagueRootUrl, region.ToString()) +
-                    string.Format(LeagueBySummonerUrl, Util.BuildIdsString(grp)) + LeagueEntryUrl, region
-                    ).ContinueWith(
-                        json => JsonConvert.DeserializeObject<Dictionary<long, List<League>>>(json.Result)
-                    )
-                ).ToList();
-
-            await Task.WhenAll(tasks);
-            return tasks.SelectMany(task => task.Result).ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
+            var json = await requester.CreateGetRequestAsync(LeagueRootUrl +
+                                                       string.Format(LeagueBySummonerUrl, summonerId), region
+            );
```

Comment:
let's indent this call properly

Reply:
Ah yep forgot to reindent the file. My bad.